src/components/BuildMenu/BuildMenuEntry/BuildMenuEntry.tsx

```diff
@@ -95,7 +96,7 @@ const BuildingMenuEntry = ({
                 remove
               </button>
               <button className="building-menu__modal-close" onClick={() => setModalOpen(false)}>
-                close
+              ⭕
```

Comment:
❌?

Reply:
I thought the rounded icon is less agressive than the x but indicates the same purpose.